content/docs/ops/setup/istioctl/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: istioctl auto-completion
+description: Istio includes a helpful script that enables auto-completion for the currently available istioctl commands.
+weight: 20
+keywords: [istioctl,bash,zsh,shell,command-line]
+---
+
```

Comment:
Add and overview section here that for now just gives a very short (e.g., 1 paragraph) summary of what istioctl is and link to the reference doc.

Then put all the auto-completion stuff under another heading after the overview.

Reply:
@frankbu will do.